README.md

```diff
@@ -54,6 +54,22 @@ To build Gaffer run `mvn clean install -Pquick` in the top-level directory. This
 
 The Accumulo store needs to run on a Hadoop cluster with Accumulo installed. Instructions for installing Gaffer on Accumulo can be found on the [wiki](https://github.com/gchq/Gaffer/wiki/Accumulo-Store-User-Guide#accumulo-set-up).
 
+### Inclusion in other projects
+
+Gaffer is hosted on Maven Central and can easily be incorporated into your own projects.
```

Comment:
Perhaps we could put a link to: https://mvnrepository.com/search?q=uk.gov.gchq.gaffer